client/ios/NetBirdSDK/client.go

```diff
@@ -0,0 +1,223 @@
+package NetBirdSDK
```

Comment:
Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns. https://go.dev/blog/package-names

Reply:
I know but for this package I changed the name to that format as this is how the gomobile then names the framework on Swift and I followed the Swift convention as it seemed to be the lesser problem. The iOS package in go will never be referenced but the Swift package will be referenced throughout the whole app.